packages/fxa-settings/src/components/Settings/BentoMenu/index.tsx

```diff
@@ -31,8 +31,8 @@ export const BentoMenu = () => {
   const ftlMsgResolver = useFtlMsgResolver();
 
   const bentoMenuTitle = ftlMsgResolver.getMsg(
-    'bento-menu-title',
-    'Firefox Bento Menu'
+    'bento-menu-title-2',
+    'Mozilla Bento Menu'
```

Comment:
I'm not sure about this one. I wonder if something like "Mozilla Products Menu" might be more accurate/helpful 🤔

Reply:
Maybe we should rethink this UI element later? I feel like people probably don't utilize it very much. Having the products immediately visible seems better than having them hidden on a menu that has low affordance for 'product links'.

Reply:
I've filed a followup for the revisiting the UX, and I'll bring up the phrasing for review. I agree that 'bento' maybe isn't the best term.